docs/sources/setup-grafana/configure-security/secretscan.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
```

Comment:
@Jguer, is this a new file (it appears to be) - if so, we don't need aliases. We only add an alias when the topic moves.